unicodetools/org/unicode/tools/CompareUnicodeSets.java

```diff
@@ -0,0 +1,17 @@
+package org.unicode.tools;
+
```

Comment:
Another quick and dirty tool. This one was for use in CLDR exemplar analysis.

Reply:
(so it probably does not belong in a PR about emoji data, but not a big deal)